### PR TITLE
Improve documentation after struggling to set up and test product-det…

### PR DESCRIPTION
**Pull request by JohanLorenzo**
…ails locally

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,12 +1,72 @@
 Ship It API and Frontend
 ========================
 
-See ``api`` and ``frontend`` READMEs for more details on each.
```
Comment (JohanLorenzo) [inline README.rst:4]: These files either don't exist or are plain empty.

File: README.rst [hunk 2]
```diff
@@ -1,12 +1,72 @@
...
+To rebuild product-details
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+product-details rely on a ``pulse queue <https://github.com/mozilla-releng/shipit/blob/df379442c32baa7931767b058840bbb293135010/api/src/shipit_api/admin/api.py#L229>``_, which makes local test tricky.
+This pulse queue is then consumed by ``worker.py <https://github.com/mozilla-releng/shipit/blob/df379442c32baa7931767b058840bbb293135010/api/src/shipit_api/admin/worker.py#L42>``_. Although, there's a
+way to by-pass the need for a pulse queue.
+
+1. ``docker-compose run api bash``
+
+2. ``shipit_rebuild_product_details --database-url="postgres://shipituser:shipitpassword@db/shipitdb" --channel dev``
```
[the remark this answers is not in the document]
Comment (JohanLorenzo) [reply README.rst:47]: You're right! I shouldn't have trusted blindly my shell history 😅 Fixed!

File: README.rst [hunk 3]
```diff
@@ -1,12 +1,72 @@
...
+Troubleshooting
+~~~~~~~~~~~~~~~
+
+**"Are you connected to the VPN?"**
+
+You may see the error message ``Error contacting Shipit backend: Error: Network Error. Are you connected to the VPN?``. Be warned this is just a generic message and you don't have to
+be connected to the VPN when locally running the instance. This error message can be misleading. Always look at the Firefox developer console, on the "network" tab to check what error
+message the API actually returned. If you end up getting a CORS error, then redo the "first setup"
+
+**docker-compose up just don't manage to start properly**
```
[the remark this answers is not in the document]
Comment (JohanLorenzo) [reply README.rst:63]: Oops! Fixed!

**Review by camd — APPROVED** [review]
Thanks so much for doing this!

**Pull request merged**